Source/WebKit/NetworkProcess/cache/NetworkCacheStorage.h

```diff
@@ -198,8 +204,10 @@ class Storage : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Storage,
     HashSet<std::unique_ptr<ReadOperation>> m_activeReadOperations;
     WebCore::Timer m_readOperationTimeoutTimer;
 
+    Lock m_activitiesLock;
+    HashCountedSet<WriteOperationIdentifier> m_writeOperationActivities WTF_GUARDED_BY_LOCK(m_activitiesLock);
```

Comment:
What is this used for? I see us adding and removing from it but didn't see any other usage?

Reply:
`remove()` returns true if count drops to 0 and item is removed.